src/assets/js/customizer-controls/integration-control/style.css

```diff
@@ -211,4 +211,8 @@ a.mo-integration-form-remove, a.mo-integration-delete {
 
 .customize-control textarea {
     width: 100%;
+}
+
+a.mo-webhook-header-fields-remove {
```

Comment:
@ShoboySnr is this css used anywhere?

Reply:
@w3guy yes it is. the button that removes the fields row when appended